tests/spec/core/xref-spec.js

```diff
@@ -197,6 +197,24 @@ describe("Core — xref", () => {
     expect(link.title).toEqual("Error: Linking an ambiguous dfn.");
   });
 
+  it("disambiguates response based on context", async () => {
+    // https://github.com/w3c/respec/pull/1750
+    const body = `
+      <p data-cite="webidl"><a id="one">object</a></p>
+      <p data-cite="html"><a id="two">object</a></p>
```

Comment:
`
<p data-cite="html">
   <a id="three" data-cite="OTHER">object</a> 
   <a>object</a>
</p>
`

Reply:
updated test case.

Reply:
`OTHER` is intended to fail, right?

Reply:
And maybe can add a expected fail case.

Reply:
```HTML
<a id="five" data-cite="">object</a>
<a id="six" data-cite="NOT-FOUND">object</a>
```